src/components/articles-list/articles-list.jsx

```diff
@@ -0,0 +1,49 @@
+import { useState, useEffect } from "react";
+import { getArticles } from "../../utils/api";
+import ArticleCard from "../article-card/article-card";
+
+function ArticlesList() {
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
this should start as false and we set it to true as the first thing in the useEffect.